pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
The only one that needs longdouble, though, is F0 (and that only in *extreme* cases).  The others *never* have even close to that precision requirement.